web/themes/custom/nysenate_theme/nysenate_theme.theme

```diff
@@ -4190,6 +4189,18 @@ function nysenate_theme_preprocess_node__resolution(&$variables) {
     ->resolveAmendmentSponsors($node, $chamber);
 
   $variables['active_amend_url'] = $node->toUrl('canonical', ['absolute' => TRUE])->toString();
+
+  // Sponsor.
+  $sponsor = NULL;
+  if ($node->hasField('field_ol_sponsor') && !empty($node->field_ol_sponsor->target_id)) {
```

Comment:
If the `if()` condition is `FALSE`, then `$variables['sponsor_link']` will be undefined.